scripts/release/mule/package/linux/amd64/docker/package.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
+echo
+
+OS_TYPE="$1"
+ARCH="$2"
+WORKDIR="$3"
+
+if [ -z "$OS_TYPE" ] || [ -z "$ARCH" ] || [ -z "$WORKDIR" ]; then
+    echo OS, ARCH and WORKDIR variables must be defined.
+    exit 1
+fi
+
+BRANCH=$(./scripts/compute_branch.sh)
+export BRANCH
+CHANNEL=$(./scripts/compute_branch_channel.sh "$BRANCH")
+export CHANNEL
+
+TARBALLS="./tmp/node_pkgs/$OS_TYPE/$ARCH"
+FULLVERSION=$(./scripts/compute_build_number.sh -f)
+export FULLVERSION
+
+ALGOD_INSTALL_TAR_FILE="$TARBALLS/node_${CHANNEL}_${OS_TYPE}-${ARCH}_${FULLVERSION}.tar.gz"
+
+if [ -f "$ALGOD_INSTALL_TAR_FILE" ]; then
+    echo "using install file $ALGOD_INSTALL_TAR_FILE"
+else
+    echo "error, $ALGOD_INSTALL_TAR_FILE does not exist"
+    exit 1
+fi
+
+INPUT_ALGOD_TAR_FILE="temp_install.tar.gz"
+CHANNEL_VERSION="${CHANNEL}_${FULLVERSION}"
+PKG_DIR="algod_pkg_$CHANNEL_VERSION"
+DOCKER_EXPORT_FILE="algod_docker_export_$CHANNEL_VERSION.tar.gz"
+DOCKER_PKG_FILE="algod_docker_package_$CHANNEL_VERSION.tar.gz"
+DOCKER_IMAGE="algorand/algod_$CHANNEL_VERSION:latest"
+DOCKERFILE=./docker/build/algod.Dockerfile
+START_ALGOD_FILE=./docker/release/start_algod_docker.sh
+ALGOD_DOCKER_INIT=./docker/release/algod_docker_init.sh
```

Comment:
it would be nice to have all of these in double quotes.

Reply:
Changed